### ISSUE TITLE
연도 범위에 대하여

### ISSUE THREAD

**Issue by cheezafizz** [swsnu/swppfall2020#20]
스켈레톤을 보면 python run.py 2001 2018을 기준으로 작성된 것을 알 수 있는데,
혹시 argument로 2001, 2018이 아니라 다른 값이 들어올 때도 정상적으로 작동해야하나요?

**Comment by Algy** [comment]
네, 해당 범위가 아닌 더 좁은 범위나, 다른 연도의 html이 `babydata/`에 저장되어 (e.g. 1994.html) 2001년 이전에 해당하는 범위가 들어올 수 있다는 가정 하에 코드를 작성해주시길 바랍니다. (이는 현 스켈레톤 코드 상에서 `year1`, `year2`를 파라메터로 하고 특정 `year`에 specific한 코드를 넣지 않았다면 자연스럽게 이루어지는 동작입니다.)

**Comment by cheezafizz** [comment]
그렇다면 rank change를 계산하는 것은 argument로 들어온 두 연도 사이의 데이터만 있다고 가정을 하고 계산하는 것이 맞나요?
아니면 첫 연도의 경우 이전 연도를 따로 fetch해서 구해줘야하나요?
전자가 맞다면 항상 첫 연도의 모든 rank change는 0이 되는 것이 맞나요?

**Comment by Algy** [comment]
인자로 두 연도 범위의 데이터만 고려하면 됩니다. 그렇기에 인자로 들어온 첫 연도에 해당하는 모든 레코드의 rank change는 전년도 데이터가 없는 것으로 가정이 되기 때문에 항상 `None`이 됩니다.

**Comment by kooyunmo** [comment]
예를 들어 `python run.py 2017 2018`을 한 경우 2017년에 해당하는 데이터들의 rank_change는 모두 `None`이면 됩니다.

**Comment by cheezafizz** [comment]
늦은 시간에도 답변 감사합니다.
